src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Scope/ScopeControllerTests.cs

```diff
@@ -11,16 +11,17 @@ public class ScopeControllerTests : ScopeControllerTestsBase
         public async Task GetCommPkgsInProject_AsViewer_ShouldGetCommPkgsInProject()
         {
             // Act
-            var commPkgs = await ScopeControllerTestsHelper.GetCommPkgsInProjectAsync(
+            var commPkgSearchResult = await ScopeControllerTestsHelper.GetCommPkgsInProjectAsync(
                 UserType.Viewer,
                 TestFactory.PlantWithAccess,
                 TestFactory.ProjectWithAccess,
                 "CommPkgNo");
 
             // Assert
-            var commPkg1 = commPkgs.First();
-            var commPkg2 = commPkgs.Last();
-            Assert.AreEqual(2, commPkgs.Count);
+            var commPkg1 = commPkgSearchResult.CommPkgs.First();
```

Comment:
Changes as this in an integration test "proof" the breaking change mentioned in controller.
We should discuss how to solve it ..